docs/automated-migration.md

```diff
@@ -4,6 +4,8 @@ hide:
 - toc
 ---
 
+<!-- markdownlint-disable MD029 -->
```

Comment:
The numbered list item in this page are not next to each other. So we wrongly get a warning that our list should start at 1.

https://github.com/DavidAnson/markdownlint/blob/v0.36.0/doc/md029.md